docker-compose.test.py2.yml

```diff
@@ -213,7 +213,7 @@ services:
       timeout: 5s
       interval: 5s
       retries: 5
-      test: "curl -k localhost:4566|grep running"
+      test: "curl -k localstack:4566|grep running"
```

Comment:
do you we need to change to `localstack`?

Reply:
We set the `HOSTNAME_EXTERNAL=localstack` and `HOSTNAME=localstack` so I think that we should use localstack here (locally when I test with `docker-compose` the log is `Unable to find forwarding rule for host "localhost:4566"`)